helm/cfgov/templates/tests/smoke-test.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "smoke-test-job"
+  labels:
```

Comment:
Just use the `app.labels` template (see other tests), no need to add them each in individually.